### DISCUSSION TITLE
Template Directories

### DISCUSSION THREAD

**Discussion by barraIhsan** [twpayne/chezmoi#4089] [Q&A] [answered]
sorry if i miss something on the docs, but can I do something like
`{{ if eq .chezmoi.username "barra" -}}` but on directories?
I have `private_dot_ssh` that will be placed on `~/.ssh`. The problem is that if I use my dotfiles on my friend's machine (private ssh key on bitwarden and fetched using `rbw`) or if someone else on the internet want to use my dotfiles, I wouldn't want my ssh config, and my ssh pub key on their computer.

Yes, I could've put `{{ if eq .chezmoi.username "barra" -}}` in every single file inside that, but
1. Maybe it would leave an empty directory
2. Repetitive

Thanks!

**Comment by twpayne** [comment, marked as answer]
Use `.chezmoiignore`, which is a template, to ignore the files that you don't want on your friend's machine.

**Reply by barraIhsan** [reply]
ahhh, like this? Thanks!
```
{{ if not (eq .chezmoi.username "barra") -}}
.ssh/
{{ end }}
```

oh is there any way to define that it's my computer without username checking (cuz people can use the same username)

**Reply by twpayne** [reply]
> ahhh, like this?

Yes.



> oh is there any way to define that it's my computer without username checking

Maybe use the `.chezmoi.hostname` template variable or one of the other variables that a printed by `chezmoi data`?

**Reply by barraIhsan** [reply]
ahh ok, or I'd probably just leave it like that or use my own variables on `.chezmoi.toml`, thanks tho!
